src/pages/Home/Home.page.tsx

```diff
@@ -76,6 +97,17 @@ const HomePage = () => {
                     </div>
                 </Card>
 
+                <Card title="RSVP Status" className="xl:col-span-5">
+                    <span className="flex flex-col gap-2">
+                        <p className="text-[#333] text-sm">
+                            RSVP status: <b>{rsvpStatus}</b>
+                        </p>
+                        <button className="p-2 bg-white rounded-md">
+                            Not able to make it?
+                        </button>
```

Comment:
We are in the process of making the waitlist, so I wouldn't put this here just yet.

Reply:
Sounds good, removed in the newest commit